include/votca/tools/cubicspline.h

```diff
@@ -99,6 +99,20 @@ class CubicSpline : public Spline
     void AddToFitMatrix(matrix_type &A, double x,
             int offset1, int offset2=0, double scale=1);
 
+    /**
+     * \brief Add a point (one entry) to fitting matrix
+     * \param pointer to matrix
+     * \param value x
+     * \param offsets relative to getInterval(x)
+     * \param scale1 parameters for terms "A,B,C,D"
```

Comment:
Maybe add [in] [out] brackets next to the param keyword to indicate if the param is being changed?

Reply:
I now have added the brackets